docs/section11/Rule_11-9.md

```diff
@@ -0,0 +1,31 @@
+# Service_Water_Heating - Rule 11-9
+**Schema Version:** 0.0.38  
+
+**Mandatory Rule:** FALSE
+
+**Rule ID:** 11-9
+
+**Rule Description:** "The baseline system must be sized according to Standard 90.1 2019, Section 7.4.1."  
+
+**Rule Assertion:** Applicable / not applicable
+
+**Appendix G Section Reference:** Table G3.1 #11, proposed column, a & b
+
+**Evaluation Context:** B-RMD each SHW type
```

Comment:
Recommend avoiding the use of SHW type language as the meaning is not fully clear